### PR TITLE
Update hybrid-modern-auth-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
updated ADFS to AD FS

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/925

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: Enterprise/hybrid-modern-auth-overview.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ Verify and check these items off your list before you continue:
   |Exchange ActiveSync clients (e.g., iOS11 Mail)  <br/> |Exchange ActiveSync  <br/> |For Exchange ActiveSync clients that support modern authentication, you must recreate the profile in order to switch from basic authentication to modern authentication.  <br/> |
 
 - **General prerequisites**
-  - If you use ADFS, you should have Windows 2012 R2 ADFS 3.0 and above for federation
+  - If you use AD FS, you should have Windows 2012 R2 AD FS 3.0 and above for federation
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md:154]:
```suggestion
  - If you use AD FS, you should have Windows 2012 R2 AD FS 3.0 and above for federation.
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ Verify and check these items off your list before you continue:
...
   - Your identity configurations are any of the types supported by Azure AD Connect (such as password hash sync, pass-through authentication, on-premises STS supported by Office 365, et cetera.)
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md:155]:
```suggestion
  - Your identity configurations are any of the types supported by Azure AD Connect, such as password hash sync, pass-through authentication, and on-premises STS supported by Office 365.
```

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks.

**Pull request merged**